doc/TOOLBOX.md

```diff
@@ -0,0 +1,53 @@
+# Toolbox testing guide
+
+## Before You Run It
+
+### Requirements
+
+* RHEL 7/8 machine
+* docker/podman installed on RHEL machine
+* sshd installed and configured on RHEL machine
+* Toolbox image pulled on RHEL machine
+* two 3scale tenants and related access tokens
```

Comment:
This is optional, if not supplied and logged in to openshift, brand new tenants are created.

Reply:
I don't see this sentence as contradiction. Toolbox tests require 2 tenants with related access tokens. It doesn't matter if they are created by magic in testsuite, they are from different OCPs or even from SaaS. As it is mentioned below, `threescale` section is optional.

Reply:
It's in section `Before you run it (tests)`

same as destination_endpoint and destination_provider_key are optional.